tests/unit/testHandleUnknownAsError.js

```diff
@@ -14,59 +14,32 @@
  * limitations under the License.
  */
 
-var test = require('tape');
+var assert = require('assert');
 var ErrorMessage = require('../../src/classes/error-message.js');
 var handleUnknownAsError = require('../../src/error-handlers/unknown.js');
-var errorClassParsingUtils = require('../../src/error-parsing-utils.js');
 
-test(
-  'Given variable inputs and input-types in the handleUnknownAsError handler'
-  , function ( t ) {
-
-    var em = new ErrorMessage();
-
-    t.plan(7);
-
-    t.doesNotThrow(
-      handleUnknownAsError.bind(null, undefined, em)
-      , undefined
-      , "Should not throw when given undefined"
-    );
-
-    t.doesNotThrow(
-      handleUnknownAsError.bind(null, null, em)
-      , undefined
-      , "Should not throw when given null"
-    );
-
-    t.doesNotThrow(
-      handleUnknownAsError.bind(null, {}, em)
-      , undefined
-      , "Should not throw when given an object"
-    );
-
-    t.doesNotThrow(
-      handleUnknownAsError.bind(null, [], em)
-      , undefined
-      , "Should not throw when given an array"
-    );
-
-    t.doesNotThrow(
-      handleUnknownAsError.bind(null, new Error("test"), em)
-      , undefined
-      , "Should not throw when given an error"
-    );
-
-    t.doesNotThrow(
-      handleUnknownAsError.bind(null, 1.3, em)
-      , undefined
-      , "Should not throw when given a number"
-    );
-
-    t.doesNotThrow(
-      handleUnknownAsError.bind(null, "string_test", em)
-      , undefined
-      , "Should not throw when given a string"
-    );
-  }
-);
+describe('handleUnknownAsError behvaiour under varying input', function () {
```

Comment:
I think this would just make a single file more complex. These are all separate modules with different sets of concerns and inputs